config.lua

```diff
@@ -29,6 +29,10 @@ statusTimeout = 5000
 replaceKickOnLogin = true
 maxPacketsPerSecond = 25
 
+-- Store configuration
+storeImagesUrl = "http://static.tibia.solutions/store/"
```

Comment:
Tibia is the word you should remove, not .com.
Tbh for now just use the ip variable and then /store/ or smth xD

Remember to add http://